target/linux/rockchip/Makefile

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2012-2020 OpenWrt.org
+# Copyright (C) 2017 LEDE project
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+ARCH:=arm
+BOARD:=rockchip
+BOARDNAME:=Rockchip RK33xx
+FEATURES:=ext4 audio usb usbgadget display gpio fpu rootfs-part boot-part
+SUBTARGETS:=rk3399
+
+KERNEL_PATCHVER=5.4
+
+define Target/Description
+	Build firmware image for Rockchip RK3399 SoC devices.
+endef
+
+include $(INCLUDE_DIR)/target.mk
+
+DEFAULT_PACKAGES += \
+	uboot-envtools kmod-usb-hid kmod-fs-vfat kmod-nls-cp437 \
```

Comment:
Is vfat needed as you already have fat support in kernel config?
Same goes for nls packages and usb-hid...

Reply:
Yep, good idea.